java/README.md

```diff
@@ -135,6 +135,14 @@ Additionally, usages of `STALE_FLAG` will be removed as if the enum itself had b
 
 Finally, the setting `linkURL` in the properties file is to provide a URL describing the Piranha tooling and any custom configurations associated with the codebase. 
 
+Another top-level optional field is `unnecessaryTestMethodProperties`. Within that, there is an array of JSON objects, 
+having the fields `methodName`, `argumentIndex`, `receiverType`, `returnType` and `isStatic`.
+
+In practice, often the refactored code produced by `piranha` results in unnecessary method invocations. For instance, 
```

Comment:
let us get rid of the term `often` :). How about "The refactored code may contain unnecessary method invocations".